CMakeLists.txt

```diff
@@ -261,7 +261,7 @@ if (XCB_XFIXES_FOUND)
     target_link_libraries(screengrab ${XCB_XFIXES_LIBRARY})
 endif()
 
-target_link_libraries(screengrab qkeysequencewidget Qt5::Widgets KF5::WindowSystem ${X11_LIBRARIES})
+target_link_libraries(screengrab qkeysequencewidget Qt5::Widgets KF5::WindowSystem Qt5::X11Extras Qt5::Network ${X11_LIBRARIES})
```

Comment:
Please add a comment, why this is needed right now - the only module which need network and i guess x11extras should be the now broken upload module - which is work in progress.

Reply:
Done :)